### PR TITLE
change default nulable type fields from maybe to maybeNull

### PR DESCRIPTION
**Pull request by pvpshoot**
I think this can be help
#51

### CODE REVIEW

**Comment by brobertsUPS** [comment]
Hey @pvpshoot, thanks for getting to this. It dropped off my radar for a bit. I am less familiar with this code base but I wonder if we need to add tests for this?

**Comment by elie222** [comment]
The examples likely need updating too

**Comment by elie222** [comment]
Which are serving as a sort of test suite right now

**Comment by pvpshoot** [comment]
ok, i will try to add some test cases

**Comment by pvpshoot** [comment]
It was rathe hard, tests don't run, examples don't build according readme. i replaced cases wit type.maybeNull, i think all must be good.
please check @elie222 @brobertsUPS

**Comment by antondomratchev** [comment]
I was having a similar issue as #51. My work around was to overload the property with a maybeNull in the Model where needed. I think this fix will make the models compatible with Graphql's default type of null.
Great PR @pvpshoot

File: generator/generate.js [hunk 1]
```diff
@@ -300,16 +300,16 @@ ${generateFragments(name, primitiveFields, nonPrimitiveFields)}
...
           )
         case "OBJECT":
           return wrap(
             handleObjectFieldType(fieldName, fieldType),
             useMaybe,
-            "types.maybe(",
+            "types.maybeNull(",
             ")"
-          )
+
```
Comment (chrisdrackett) [inline generator/generate.js]:
```suggestion
)
```

Comment (chrisdrackett) [reply generator/generate.js]: missing a closing `)` for `wrap`

**Comment by chrisdrackett** [comment]
@pvpshoot I think you are missing a `)` but other than that this works when run in @brobertsUPS and my project.

Thanks for the PR!

**Comment by elie222** [comment]
Would like to get this one but have some merge conflicts unfortunately

**Pull request merged**